ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/execution/StaticQueryExecutorTest.java

```diff
@@ -51,57 +52,66 @@
 
     @Test
     public void shouldThrowExceptionIfConfigDisabled() {
+      // Given:
+      final ConfiguredStatement<Query> query = ConfiguredStatement.of(
+          PreparedStatement.of("SELECT * FROM test_table;", mock(Query.class)),
```

Comment:
Shouldn't this have a `WHERE ROWKEY=...` since it is a pull query?

Reply:
No need. It's not reaching any code that would validate for that.